packages/app-project/src/screens/ProjectAboutPage/components/AboutDropdownNav/AboutDropdownNav.spec.js

```diff
@@ -1,57 +1,23 @@
-import { shallow } from 'enzyme'
-import { AboutDropContent, AboutDropdownNav } from './AboutDropdownNav'
-import AboutNavLink from '../AboutNavLink'
-
-describe('Component > AboutDropdownNav', () => {
-  let wrapper
-
-  const router = {
-    query: {
-      owner: 'test-owner',
-      project: 'test-project'
-    }
-  }
-
-  before(() => {
-    wrapper = shallow(<AboutDropdownNav aboutNavLinks={[]} router={router} />)
-  })
-
-  it('should render without crashing', () => {
-    expect(wrapper).to.be.ok()
-  })
-})
-
-describe('Component > AboutDropContent', () => {
-  let wrapper
-
-  const router = {
-    query: {
-      owner: 'test-owner',
-      project: 'test-project'
-    }
-  }
-
-  before(() => {
-    wrapper = shallow(<AboutDropContent aboutNavLinks={[]} router={router} />)
-  })
-
-  it('should render without crashing', () => {
-    expect(wrapper).to.be.ok()
-  })
-
-  it('should always render at least two links: Research and The Team', () => {
-    const links = wrapper.find(AboutNavLink)
-    expect(links).to.have.lengthOf(2)
+import { render, fireEvent } from '@testing-library/react'
+import * as stories from './AboutDropdownNav.stories'
+
+describe('Component > AboutDropdownNav', function () {
+  const { Default, MoreLinks } = stories
+
+  it('should always render at least two links: Research and The Team', function () {
+    window.scrollTo = () => {} // this a side effect behavior of clicking a Grommet DropButton
```

Comment:
I would recommend that this get setup in a before hook and cleaned up in an after hook so we can restore any default behavior JSDOM might be set up for this. I think you should be able to use sinon's stub with `callsFake` and then `restore` or you can manually do it by storing the original behavior in a scoped variable to the describe block. Here's an example of that in another test: https://github.com/zooniverse/front-end-monorepo/blob/28b5eeba2fd7eeabfb669dbd8e274314eefd3e17/packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/ExpertOptions/ExpertOptionsContainer.spec.js#L33-L52